lms/djangoapps/instructor/tests/test_api.py

```diff
@@ -15,6 +15,7 @@
 import pytest
 import pytz
 from boto.exception import BotoServerError
+from dateutil.zoneinfo import gettz
```

Comment:
Why not the original `dateutil.tz`?  According to the [python-dateutil docs](https://dateutil.readthedocs.io/en/stable/zoneinfo.html#dateutil.zoneinfo.gettz), the method you have here is deprecated.

Reply:
Updated.
Two tests were failing when I first created this PR that's why I tried to use this method.
But now tests are passing even after removing this method. So now using the original `dateutil.tz`
as you suggested.